Sprint-1/fix/median.js

```diff
@@ -3,9 +3,15 @@
 // If you're in the Sprint-1 directory, you can run `npm test -- fix` to run the tests in the fix directory
 
 function calculateMedian(list) {
+  list.sort((a,b)=> a-b);
```

Comment:
Array is passed to a function ***by reference***  (instead of ***by value***). This implies when an array is passed to a function via parameter, the array is **shared** by the caller and the ***callee*** (the function being called).

To avoid modifying the caller's array, a common practice is to make a clone of the array (that's passed as a parameter) and modify the clone in the callee instead.

If you are unfamiliar with this concept, you can try asking ChatGPT "Explain how array is passed by reference in JS".